test/general/__init__.py

```diff
@@ -1,8 +1,8 @@
 from argparse import Namespace
-from typing import Optional, Tuple, Type
+from typing import List, Optional, Tuple, Type, Union
 
-from BaseClasses import MultiWorld, CollectionState
-from worlds.AutoWorld import call_all, World
+from BaseClasses import CollectionState, MultiWorld
+from worlds.AutoWorld import World, call_all
```

Comment:
It's not a big issue for this file, but in the future, i would prefer to not touch import blocks that don't need changing in any core files (unless you try to clean up a big mess, in which case we might have to reason about import order). It's easy to create merge conflicts and doesn't really serve any purpure.